lib/services.identity.js

```diff
@@ -162,6 +169,18 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     ensureAuthenticated,
     validate({query: 'services.identity.getIdentitiesQuery'}),
     function(req, res, next) {
+      // TODO: support URL query parameters
+      var q = {
+        'identity.sysStatus': 'active'
+      };
+      brIdentity.getAll(null, q, function(err, records) {
```

Comment:
This call has no authentication check. Anyone can hit this endpoint and get all of the active identities stored in the database. Change `null` to `req.user.identity`.